cmd/bootnode/main.go

```diff
@@ -11,13 +11,31 @@ import (
 	"github.com/ethereum/go-ethereum/p2p/discv5"
 )
 
+type nursery []*discv5.Node
+
+func (f *nursery) String() string {
+	return "discv5 nursery nodes"
+}
+
+// Set parses single topic.
+func (f *nursery) Set(value string) error {
+	n, err := discv5.ParseNode(value)
+	if err != nil {
+		return err
+	}
+	*f = append(*f, n)
```

Comment:
No check of duplicates?

Reply:
tbh i don't know if it is worth adding that check

Reply:
Yeah, it's only a local use case. So it's ok. Only my warning bell always rings when a list is used as a set. 😁